### PR TITLE
Update impressions in all readmes

### PR DESCRIPTION
**Pull request by weshaggard**
Update impression pixel links.

PTAL @scbedd

cc @Azure/azure-sdk-eng

### CODE REVIEW

**Comment by pakrym** [comment]
Do these even work? We are not sending Cache control headers in response:
```
curl.exe -s -D - https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Feng%2FMicrosoft.Build.CentralPackageVersions%2FReadme.png
HTTP/1.1 200 OK
Content-Length: 119
Content-Type: image/png
Server: Kestrel
Request-Context: appId=cid-v1:9c7ead92-5536-4007-b1c9-62163735e22b
Strict-Transport-Security: max-age=2592000
X-Powered-By: ASP.NET
Set-Cookie: ARRAffinity=ebe4e61f36f113f44f7147faddf771d3d74e9d0337e98a1b452f32623cdd9dcc;Path=/;HttpOnly;Domain=azure-sdk-impressions.azurewebsites.net
Date: Thu, 31 Oct 2019 21:26:24 GMT
```

And github caches the image: 

image

Returning it with  `Cache-Control: public, max-age=31536000`

```
curl.exe -s -D - https://camo.githubusercontent.com/f767f1d14a8e06658557c6fb22cc627f575318c1/68747470733a2f2f617a7572652d73646b2d696d7072657373696f6e732e617a75726577656273697465732e6e65742f6170692f696d7072657373696f6e732f617a7572652d73646b2d666f722d6e657425324673646b253246617070636f6e66696775726174696f6e253246417a7572652e4170706c69636174696f6e4d6f64656c2e436f6e66696775726174696f6e253246524541444d452e706e67
HTTP/1.1 200 OK
Content-Type: image/png
Cache-Control: public, max-age=31536000
Content-Security-Policy: default-src 'none'; img-src data:; style-src 'unsafe-inline'
Server: github-camo (876de43e)
Strict-Transport-Security: max-age=31536000; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Xss-Protection: 1; mode=block
X-GitHub-Request-Id: A3D6:3C51:AF044:D5AE5:5DBB51BB
Content-Length: 119
Accept-Ranges: bytes
Date: Thu, 31 Oct 2019 21:27:39 GMT
Via: 1.1 varnish
Age: 13
Connection: keep-alive
X-Served-By: cache-sea4476-SEA
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1572557259.124779,VS0,VE0
Vary: Accept-Encoding
X-Fastly-Request-ID: 0924995eddd2edbec6c7758f3dd6a3255b4701f2
Timing-Allow-Origin: https://github.com
```

File: sdk/storage/Azure.Storage.Blobs/swagger/readme.md [hunk 1]
```diff
@@ -1393,3 +1393,6 @@ directive:
     delete $.properties.Permission;
     $.required = ["StartsOn", "ExpiresOn", "Permissions"];
 ```
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fstorage%2FAzure.Storage.Blobs%2Fswagger%2Freadme.png)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/swagger/readme.md]: This is a swagger spec - please remove.

File: sdk/storage/Azure.Storage.Common/swagger/Generator/readme.md [hunk 2]
```diff
@@ -47,3 +47,6 @@ scope-azure-track2-csharp-generator/emitter:
 output-artifact: azure-track2-csharp-generator-code
 
 ```
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fstorage%2FAzure.Storage.Common%2Fswagger%2FGenerator%2Freadme.png)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Common/swagger/Generator/readme.md]: This is a swagger spec - please remove.

File: sdk/storage/Azure.Storage.Files.DataLake/swagger/readme.md [hunk 3]
```diff
@@ -222,4 +222,6 @@ directive:
     $.FileSystemList["x-az-public"] = false;
     $.PathList["x-az-public"] = false;
     $.Path["x-az-public"] = false;
-```
\ No newline at end of file
+```
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fstorage%2FAzure.Storage.Files.DataLake%2Fswagger%2Freadme.png)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.DataLake/swagger/readme.md]: This is a swagger spec - please remove.

File: sdk/storage/Azure.Storage.Files.Shares/swagger/readme.md [hunk 4]
```diff
@@ -749,3 +749,6 @@ directive:
   transform: >
     $.put.parameters[3]["x-ms-enum"].name = "ShareFileRangeWriteType";
 ```
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fstorage%2FAzure.Storage.Files%2Fswagger%2Freadme.png)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.Shares/swagger/readme.md]: This is a swagger spec - please remove.

File: sdk/storage/Azure.Storage.Queues/swagger/readme.md [hunk 5]
```diff
@@ -456,3 +456,6 @@ directive:
     $.RetentionPolicy["x-ms-client-name"] = "QueueRetentionPolicy";
     $.RetentionPolicy.xml = { "name": "RetentionPolicy"};
 ```
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fstorage%2FAzure.Storage.Queues%2Fswagger%2Freadme.png)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Queues/swagger/readme.md]: This is a swagger spec - please remove.

File: tools/legacy/ScriptBackup/autogenForSwaggers/readme.md [hunk 6]
```diff
@@ -63,4 +63,4 @@ Optional `dotNet` properties:
 |`dotNet.commit`   |a specification commit id       |current commit id                                   |`undefined`                         |
 |`dotNet.autorest` |an AutoRest package             |current AutoRest                                    |`undefined`                         |
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions?path=azure-sdk-for-net%2F%2Ftools%2Flegacy%2FScriptBackup%2FautogenForSwaggers%2Freadme.md)
```
Comment (tg-msft) [inline tools/legacy/ScriptBackup/autogenForSwaggers/readme.md]: Probably remove?

File: sdk/postgresql/Microsoft.Azure.Management.PostgreSQL/src/README.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
 # Swagger JSON
 This is a swagger JSON built by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsrc%2FSDKs%2FPostgreSQL%2FManagement.PostgreSQL%2FREADME.png)
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fpostgresql%2FMicrosoft.Azure.Management.PostgreSQL%2Fsrc%2FREADME.png)
```
Comment (tg-msft) [inline sdk/postgresql/Microsoft.Azure.Management.PostgreSQL/src/README.md]: This is a swagger spec - please remove.

**Review by tg-msft — APPROVED** [review]
Please remove these from swagger readmes.

**Comment by scbedd** [comment]
@pakrym They definitely show impressions, when I tested I did see repeated hits from the same user, but that has to be unreliable given the information you're surfacing. Can definitely adjust the cache settings server side.

**Comment by pakrym** [comment]
We need to double check if what we think is happening is actually happening. I know we are getting some data I'm not sure we are getting the correct data with github caching images.

**Comment by pakrym** [comment]
The data we collected is very strange: 
image

Why is Java AppConfig on top? Why is neither of repo root readmes in the top?

**Review by jsquire — APPROVED** [review]
Event Hubs changes look good to me.   Apologies for the typo that got cut/pasted; that's my bad.

**Comment by weshaggard** [comment]
>Please remove these from swagger readmes.

@tg-msft do you know of any issue they cause in the swagger readmes? If not I'd like to keep them because otherwise we will have to add an exclusion for each one to not be generated again with our tools.

**Comment by tg-msft** [comment]
> tg-msft do you know of any issue they cause in the swagger readmes?

Because it's weird to put tracking pixels in source code files?  I can try to run it and see tomorrow, but I really just don't want it in a source file.

**Pull request merged**